src/main/java/com/minecolonies/inventory/InventoryField.java

```diff
@@ -0,0 +1,344 @@
+package com.minecolonies.inventory;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.item.ItemSeeds;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.ChatComponentTranslation;
+import net.minecraft.util.IChatComponent;
+import net.minecraftforge.common.util.Constants;
+
+/**
+ * The custom chest of the field.
+ */
+public class InventoryField implements IInventory
+{
+    /**
+     * NBTTag to store the slot.
+     */
+    private static final String TAG_SLOT = "slot";
+
+    /**
+     * NBTTag to store the items.
+     */
+    private static final String TAG_ITEMS = "items";
+
+    /**
+     * NBTTag to store the custom name.
+     */
+    private static final String TAG_CUSTOM_NAME = "name";
+
+    /**
+     * NBTTag to store the inventory.
+     */
+    private static final String TAG_INVENTORY = "inventory";
+
+    /**
+     * Returned slot if no slat has been found.
+     */
+    private static final int NO_SLOT = -1;
+
+    /**
+     * The inventory stack.
+     */
+    private ItemStack[] stackResult   = new ItemStack[1];
+
+    /**
+     * The custom name of the inventory.
+     */
+    private String customName         = "";
+
+    /**
+     * Updated after the inventory has been changed.
+     */
+    private boolean inventoryChanged = false;
+
+    /**
+     * Creates the inventory of the citizen.
+     *
+     * @param title Title of the inventory.
+     */
+    public InventoryField(final String title)
+    {
+        super();
+        this.customName = title;
+    }
+
+    @Override
+    public int getSizeInventory()
+    {
+        return 1;
+    }
+
+    @Override
+    public int getInventoryStackLimit()
+    {
+        return 1;
+    }
+
+    @Override
+    public void markDirty()
+    {
+        this.inventoryChanged = true;
+    }
+
+    /**
+     * Checks if the inventory has been changed and then resets the boolean.
+     * @return true if it changed.
+     */
+    public boolean hasInventoryChanged()
+    {
+        if(inventoryChanged)
+        {
+            inventoryChanged = false;
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean isUseableByPlayer(final EntityPlayer entityPlayer)
+    {
+        return true;//colony.getPermissions().hasPermission(entityPlayer, Permissions.Action.ACCESS_HUTS);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)